styles/designs/carnival/parts/_example-shapes.scss

```diff
@@ -15,8 +15,18 @@
                 )),
                 map-merge($Example__Body, (
                     _components: (
-                        $UnorderedList--FirstLevel,
-                        $OrderedList--FirstLevel,
+                        map-merge($UnorderedList--FirstLevel, (
+                            _components: (
+                                $UnorderedList--SecondLevel,
+                                $OrderedList--SecondLevel,
+                            )
+                        )),
+                        map-merge($OrderedList--FirstLevel, (
+                            _components: (
+                                $UnorderedList--SecondLevel,
+                                $OrderedList--SecondLevel,
+                            )
+                        )),
```

Comment:
I added lists to `ExampleShape` for now. They will be moved to separate shape during lists refactoring.